### PR TITLE
[MNG-6759] - Add test demonstrating the issue where the wrong reposit…

### PR DESCRIPTION
**Pull request by srdo**
…ories are used for resolving transitive dependencies

Test for https://github.com/apache/maven/pull/294.

Advice on improving these would be appreciated. It's not good to be depending on an external plugin or an external repository.

### CODE REVIEW

**Comment by srdo** [comment]
Fixed so the test now has no external dependencies.

**Comment by rfscholte** [comment]
Don't forget to add your test to `org.apache.maven.it.IntegrationTestSuite`

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6759TransitiveDependencyRepositoriesTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+/**
+ * This is a test for <a href="https://issues.apache.org/jira/browse/MNG-6759">MNG-6759</a>.
+ */
+public class MavenITmng6759TransitiveDependencyRepositoriesTest extends AbstractMavenIntegrationTestCase {
+
+    private final String projectBaseDir = "/mng-6759-transitive-dependency-repositories";
+
+    public MavenITmng6759TransitiveDependencyRepositoriesTest() {
+        super( "(,3.6.1],[3.6.3,)" );
```
Comment (rfscholte) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6759TransitiveDependencyRepositoriesTest.java]: `super( "(,3.6.2),(3.6.2,)" );` is the preferred call.

**Comment by srdo** [comment]
Thanks for letting me know about the suite. Addressed your comments.

**Pull request merged**